build/src/VersionParser.php

```diff
@@ -29,8 +31,10 @@ public static function handleCore($version) {
     return ">=$major.$minor,<$version";
   }
 
-  public static function handleContrib($version) {
-    list($core, $version) = explode('-', $version, 2);
+  public static function handleContrib($version, $result) {
+    // $result->taxonomy_vocabulary_6 is usually a term like 8.x (https://www.drupal.org/taxonomy/term/7234).
+    // Its absence indicates a semver release (or a core release).
+    list($core, $version) = empty($result->taxonomy_vocabulary_6) ? [NULL, $version] : explode('-', $version, 2);
     list($major) = explode('.', $version);
     return ">=$major,<$version";
```

Comment:
Like core, this should probably also be ">=$major.$minor,<$version" for semver contrib versions.

Reply:
I wonder why that wasnt there before. @webflo?

Reply:
This is also new for semantic versioning, since the API-compatibility-prefixed version numbers can only be 8.x-1.2, not 8.x-1.2.3.